doc/tutorials/mount.md

```diff
@@ -110,22 +127,60 @@ Let us mount a projects git configuration into the dir namespace:
+As git uses the ini format for its configuration we use the [ini plugin](src/plugins/ini/README.md).
 You can pass parameters to plugins during the mount process. This is what
 we did with `multiline=0`. Git intends the entries in its configuration
 files and the default behaviour of the ini plugin is to interpret these indented
-entries as values that span mutiple lines. With the passed parameter disable
-this behaviour and make the ini-plugin compatible to git.
+entries as values that span mutiple lines. The passed parameter disables
+this behaviour and makes the ini-plugin compatible with git configuration.
+
+Now let us see how smoothly the ini plugin sets and gets the git configuration.
+
+    # set a user name ...
+    git config user.name "Rob Banks"
+
+    # ... and read it with kdb
+    kdb get dir/git/user/name
+    Rob Banks
+
+    # set a user email with kdb ...
+    kdb set dir/git/user/email "rob.banks@dot.com"
 
-Alternatively we could have mounted another file format:
+    # and read it with git
+    git config --get user.email
+    rob.banks@dot.com
+
+
+#### Meta data ####
+
+// TODO: work on this subchapter:
+[] find nice example
+[] link to validation.md
+
+Elektra is able to store [meta data](doc/help/elektra-metadata.md) of keys, provided the format of the file that holds the configuration supports this feature.
+The ini plugin doesn't support this feature, but the [ni](src/plugins/ni/README.md) and the [dump](src/plugins/dump/README.md) plugin do.
+Meta data comes in handy if we use other plugins, than just the ones that store and retrieve data:
+
+    // TODO
+    $ sudo kdb mount example.ni user/example ni enum iconv
+    $ kdb setmeta /example/fruit check/enum "'apple', 'banana', 'grape'"
+    $ kdb set /example/vegetable apple
+    # works
+    $ kdb set /example/vegetable tomato
+    # error
 
-    $ sudo kdb mount /package.json dir/example yajl iconv from=UTF-8,to=ISO-8859-1
 
-// TODO
 #### Backends ####
+The plugins together with the configuration file form a _backend_. The backend determines how Elektra stores data below a mountpoint.
+
+You can examine every mountpoints backend by looking at the configuration below `system/elektra/mountpoints/<mountpoint>/`.
 
 ## Limitations ##
 
 One drawback of this approach is, that an application can bypass Elektra and change configuration files directly. If for example Elektra is configured to [validate](/doc/tutorial/validation.md) new configuration values before updating them, this is something you do not want to happen.
 
-Furthermore one cannot simply change the configuration file format, because it must be one the application understands. Thus one loses quite some flexibility.
+Another drawback is that mounting is static. In a previous example we mounted the `/.git/config` file into `dir/git`. Now the `dir` namespace of every directory stores the configuration below `dir/git` in this directories `/.git/config` file. And this mountpoint is the same for all users.
```

Comment:
the same for all users and directories.

Please also mention here that this is one of the reasons we make the file names longer: they need to be unique for every application.